File: ReverseWords/app/src/main/java/com/vkt4u9999/reversewords/MainActivity.java
package com.vkt4u9999.reversewords;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;



public class MainActivity extends AppCompatActivity {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        final EditText inputEditText = (EditText)findViewById(R.id.inputField);
        final TextView outputText = (TextView)findViewById(R.id.outputField);

        Button reverseButton = (Button)findViewById(R.id.reverseButton);
        reverseButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                String outputLine = inputEditText.getText().toString();
                String[] words = outputLine.split(" ");
                StringBuilder str = new StringBuilder();
                for (String x : words) {
                    str.append(" " + HalfReverse.halfReverse(x, 'x', 'l'));
                    outputText.setText(str.toString());
                }
            }
        });
    }
}

File: InstagramClone/app/src/main/java/com/vkt4u9999/instagramclone/MainActivity.kt
package com.vkt4u9999.instagramclone

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.core.view.size
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        bottom_navigation_view.setTextVisibility(false)
        bottom_navigation_view.enableItemShiftingMode(false)
        bottom_navigation_view.enableShiftingMode(false)
        bottom_navigation_view.enableAnimation(false)
        for (i in 0 until bottom_navigation_view.size){
            bottom_navigation_view.setIconTintList(i, null)
        }
    }
}